modules/extensions-spring-support/src/test/resources/models/heisenberg.json

```diff
@@ -1648,7 +1648,7 @@
                 }
               },
               "hasDynamicType": false,
-              "required": true,
+              "required": false,
```

Comment:
don't change the existing components to fit your scenario. Please create a new component instead.